Docker-files/README.md

```diff
@@ -0,0 +1,78 @@
+# KALDodocker
+
+## Overview
+KALDodocker is a Docker container designed for computational simulations, integrating LAMMPS with Python support and KALDo software. This container is tailored for CPU environments but also includes instructions for adapting it for GPU usage.
```

Comment:
It should be clear what the difference between the three versions is:
- only kaldo
- kaldo + deepmd
- kaldo + deepmd + cuda

Reply:
Ok. I will rename them. the goal was as follows.

- kalldo-dev is gpu and cpu enabled but also install the git repository for testing. This would be for developers for an environment.
- Kaldo-CPU was for kaldo with cpu only usage. It only really has kaldo and nothing else.
- Kaldo-GPU includes deepmd and other software. Zekun had helped me discuss items that would be implemented in the future, so it has some additional items.

I will go rename the files to be clearer.

Reply:
Got it. We can keep it as it is, if it makes sense. Let's just link it from the DeepMD example.